src/server.py

```diff
@@ -17,17 +17,17 @@ def plugin_module_import(name):
 
 class NotificationHandler(StreamRequestHandler):
     def handle(self):
-        length = struct.unpack('<I', self.request.recv(4))[0]
-        json_content = self.request.recv(length)
+        length = struct.unpack('<I', self.request.recv(4))[0] 
+        json_content = self.request.recv(length) 
         content = json.loads(json_content)
-        terminal = content['terminal']
-        if content['exec'] != '':
+        terminal = content['terminal'] 
```

Comment:
sadly the trailing whitespace is still present

Reply:
ok, i already kown and delete it.